### DISCUSSION TITLE
Standford EE364A: Convex Optimization - CS自学指南

### DISCUSSION THREAD

**Discussion by giscus** [PKUFlyingPig/cs-self-learning#330] [Announcements]
# Standford EE364A: Convex Optimization - CS自学指南

CS自学指南

https://csdiy.wiki/%E6%95%B0%E5%AD%A6%E8%BF%9B%E9%98%B6/convex/

**Comment by TonyZhang2004** [comment]
有没有小伙伴组队学这门课的？加QQ (17402270575)_8

**Comment by JikunR** [comment]
Steven Boyd确实牛，但是论讲课这一块，还是得看Ryan Tibshirani，循循善诱深入浅出。改天提个PR
[CMU 10-725: Convex Optimization](https://www.stat.cmu.edu/~ryantibs/convexopt/#top=)

**Reply by KindofCrazy** [reply]
请问这个的前置课程是什么 以及这个是只有notes没有视频吗

**Reply by JikunR** [reply]
就普通的线性代数微积分吧，没啥特别的。凸优化的数学推导过程不复杂，但是intuition和结论都很玄妙。
https://www.youtube.com/playlist?list=PLRPU00LaonXQ27RBcq6jFJnyIbGw5azOI

**Reply by KindofCrazy** [reply]
好的 谢谢

**Reply by ddddddavid66** [reply]
你好 我刚上这节课 还没上CMU的课 感觉这个老师 一直说 sorry ok so 其他的啥也没记得 CMU的老师会讲的更好吗

**Comment by Jinhua-Si** [comment]
有没有小伙伴组队学这门课的？加QQ：3246215301

**Comment by goahead656** [comment]
课程在youtube上的链接，youtube自动生成的字幕更准一些
https://www.youtube.com/watch?v=VNON98dKjno&list=PLoCMsyE1cvdXeoqd1hGaMBsCAQQ6otUtO&index=1

**Comment by digitzh** [comment]
Boyd、Tibshirani、中科大的视频我都看了一些，课程本身内容比较抽象，想找例题看看，一看课后作业，感到颇有难度。我认为可以先了解一下它的应用，接触一下CVX和cvxpy，从实际意义了解背后的知识，也是一个思路。课程视频讲课节奏普遍比较快，以理论部分居多(中科大的课也是讲理论部分2-5章，实践只提一下)，翻翻Boyd和Tibshirani的slides，真的超多概念...

**Comment by Mamihla** [comment]
优化课其实ucb的eecs127也还不错

**Comment by wyg129642** [comment]
请问大家，EE364B和这门EE364A有什么关系呀，有必要学吗

**Comment by lil-Journey** [comment]
自建日本节点服务器 月租5元不限流量 本人也在用 失效时可联系人工修复 qq:1165116287
